src/app/pages/Home/index.tsx

```diff
@@ -17,59 +17,49 @@ import { StakingActions } from "app/containers/BlockChain/Governance/Staking/sli
 import { GovernanceSelectors } from "app/containers/BlockChain/Governance/selectors";
 import { GovernanceActions } from "app/containers/BlockChain/Governance/slice";
 import { StakingSelectors } from "app/containers/BlockChain/Governance/Staking/selectors";
+import { SwapActions } from "app/containers/Swap/slice";
+import { BigNumber } from "ethers";
+import { SwapSelectors } from "app/containers/Swap/selectors";
+import { TokenSymbols } from "app/containers/Swap/types";
 
 export const HomePage = () => {
   useInjectReducer({ key: sliceKey, reducer: HomePageReducer });
   useInjectSaga({ key: sliceKey, saga: homePageSaga });
 
   const dispatch = useDispatch();
+  const isGettingBestSwapPath = useSelector(
+    SwapSelectors.selectIsGettingBestPath
+  );
+  const isSwapping = useSelector(SwapSelectors.selectIsSwapping);
+  const bestPath = useSelector(SwapSelectors.selectBestPath);
 
-  const handleNavigateToGovernanceClick = () => {
-    dispatch(push(AppPages.GovernancePage));
-  };
-
-  const handleNavigateToStakingClick = () => {
-    dispatch(push(AppPages.StakingPage));
-  };
-
-  const handleTestStakingClick = () => {
+  const handleGetBestSwapPathClick = () => {
     dispatch(
-      StakingActions.createLock({
-        duration: "1200",
-        date: new Date().toISOString(),
-        balance: "0",
+      SwapActions.findBestPath({
+        fromTokenSymbol: TokenSymbols.USDTe,
+        amountToGive: BigNumber.from(1),
+        toTokenSymbol: TokenSymbols.FRAX,
       })
     );
   };
-  const readyForStaking = useSelector(StakingSelectors.selectReadyForStaking);
-  const isStaking = useSelector(StakingSelectors.selectIsStaking);
-
-  const synced = useSelector(
-    GovernanceSelectors.selectSyncedProposalsWithBlockChain
-  );
-  const handleSetSyncProposals = () => {
-    dispatch(GovernanceActions.setSyncedProposalsWithBlockchain(false));
+  const handleSwapClick = () => {
+    dispatch(SwapActions.swap());
   };
 
   return (
     <>
-      <ContainedButton onClick={handleNavigateToGovernanceClick}>
-        Go To Governance
-      </ContainedButton>
-      <ContainedButton onClick={handleNavigateToStakingClick}>
-        Go To Staking
-      </ContainedButton>
-      <WalletToggle />
-      <ContainedButton loading={isStaking} onClick={handleSetSyncProposals}>
-        Synced Proposals {synced.toString()}
-      </ContainedButton>
+      {/* <Rewards /> */}
```

Comment:
remove unnecessary commented out code or please add a description why is commented out.